client/i18n/locales/english/intro.json

```diff
@@ -3955,6 +3955,17 @@
         "title": "Build a Bisection Method",
         "intro": [""]
       },
+      "workshop-merge-sort": {
+        "title": "Build a Merge Sort Algorithm",
```

Comment:
I would modify this  title in some way. Something like build (or design/implement) **the** merge sort algorithm. It seems more natural to me.

Reply:
done!